README.md

```diff
@@ -1 +1,26 @@
-# github-flow
\ No newline at end of file
+# github-flow
+
+Brief description of the project.
```

Comment:
- [ ] In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.
  - Kindly add all sections from the Microverse README.
  - The only optional section from the README file is `faq`.

| professional Readme  | Your Readme that is not professional |
| ------------- | ------------- |
|  ![Recording 2023-02-14 at 12 52 20 (5)](https://user-images.githubusercontent.com/83360636/218715072-2b2d3a49-8219-4939-b350-45cc950f7f7a.gif)	  | ![image](https://user-images.githubusercontent.com/52934128/219091958-e62eb69a-e8f9-4d60-ac68-a522504a7dbc.png)  |